src/ProjectTemplates/test/Helpers/TemplatePackageInstaller.cs

```diff
@@ -95,29 +96,29 @@ private static void InstallTemplatePackages(ITestOutputHelper output)
             }
 
             VerifyCannotFindTemplate(output, "web");
-            VerifyCannotFindTemplate(output, "razor");
+            VerifyCannotFindTemplate(output, "webapp");
+            VerifyCannotFindTemplate(output, "mvc");
             VerifyCannotFindTemplate(output, "react");
             VerifyCannotFindTemplate(output, "reactredux");
             VerifyCannotFindTemplate(output, "angular");
 
             var builtPackages = MondoHelpers.GetNupkgFiles();
-            foreach (var packagePath in builtPackages)
+            var templatePackages = builtPackages.Where(b => _templatePackages.Any(t => Path.GetFileName(b).StartsWith(t, StringComparison.OrdinalIgnoreCase)));
+            Assert.Equal(4, templatePackages.Count());
+            foreach (var packagePath in templatePackages)
             {
-                if (_templatePackages.Any(name => Path.GetFileName(packagePath).StartsWith(name, StringComparison.OrdinalIgnoreCase)))
-                {
-                    output.WriteLine($"Installing templates package {packagePath}...");
-                    RunDotNetNew(output, $"--install \"{packagePath}\"", assertSuccess: true);
-                }
+                output.WriteLine($"Installing templates package {packagePath}...");
+                RunDotNetNew(output, $"--install \"{packagePath}\"", assertSuccess: true);
             }
-            VerifyCanFindTemplate(output, "razor");
+            VerifyCanFindTemplate(output, "webapp");
             VerifyCanFindTemplate(output, "web");
             VerifyCanFindTemplate(output, "react");
         }
 
         private static void VerifyCanFindTemplate(ITestOutputHelper output, string templateName)
         {
             var proc = RunDotNetNew(output, $"", assertSuccess: false);
-            if (!proc.Output.Contains($" {templateName}"))
+            if (!proc.Output.Contains($" {templateName} "))
```

Comment:
Makes sure we don't think something like `razor` is found when actually we found `razorpage`